index.html

```diff
@@ -22,7 +25,16 @@
                 <label for="navi-toggle" class="navigation__button" id="navi-button">
                     <span class="navigation__icon"></span>
                 </label>
-                <div class="navigation__background" id="navi-background"></div>
+                <div class="menu" id="menu">
+                    <div class="bar-menu">
+                        <a onclick="closeNav()" class="closenav" href="#portfolio">Portfolio</a>
+                        <hr>
+                        <a onclick="closeNav()" class="closenav" href="#about">About</a>
+                        <hr>
+                        <a onclick="closeNav()" class="closenav" href="#contact">Contact</a>
+                        <hr>
+                    </div>
+                </div>
                 <nav class="bar" id="bar">
                     <div class="bar1"><a href="#portfolio">Portfolio</a></div>
                     <div class="bar2"><a href="#about">About</a></div>
```

Comment:
- [ **OPTIONAL** ] **RECOMENDED:** I think it would be better if you remove the duplicate **navigation list** and use the one wrapped in the `<nav>` tag for best practices. 
-  [ **OPTIONAL** ] **RECOMENDED:** For JavaScript best practices, kindly make sure you call the `closeIcon()` function in the `javascript file` and not in the `HTML` file. For that approach you can use the `addEventListener()` method.  For more help, you can watch  **[this video](https://www.youtube.com/watch?v=1iS0r238G4g)**.